met-web/src/components/engagement/view/EmailPanel.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import {
+    Grid,
+    Typography,
+    Checkbox,
+    Button,
+    TextField,
+    FormControl,
+    FormControlLabel,
+    FormHelperText,
+} from '@mui/material';
+import { EmailPanelProps } from './types';
+
+const style = {
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 700,
+    bgcolor: 'background.paper',
+    boxShadow: 10,
+    pt: 2,
+    px: 4,
+    pb: 3,
+    height: 700,
+};
+
+const label = { inputProps: { 'aria-label': 'Checkbox demo' } };
+
+const EmailPanel = ({ email, checkEmail, handleClose, updateEmail }: EmailPanelProps) => {
+    const [checked, setChecked] = useState(false);
+    const [emailFormError, setEmailFormError] = useState({
+        terms: false,
+        email: false,
+    });
+
+    const validateForm = () => {
+        const errors = {
+            terms: !checked,
+            email: email === '',
+        };
+
+        setEmailFormError(errors);
+
+        return Object.values(errors).some((isError: unknown) => isError);
+    };
+
+    const handleSubmit = () => {
+        const hasErrors = validateForm();
+
+        if (!hasErrors) {
+            checkEmail();
+        }
+    };
+
+    return (
+        <Grid container direction="row" sx={{ ...style }} rowSpacing={1}>
+            <Grid item xs={12}>
+                <Typography id="modal-modal-title" variant="h4" component="h2" sx={{ mb: 2, fontWeight: 'bold' }}>
+                    Verify your email address
+                </Typography>
+            </Grid>
+            <Grid item xs={12}>
+                <Typography id="modal-modal-header">
+                    To provide you with the best experience possible. We require you to validate your email address.
+                </Typography>
+            </Grid>
+            <Grid item xs={12}>
+                <Typography>You will receive a link to access the survey at the email address you provide.</Typography>
+            </Grid>
+
+            <Grid item xs={12}>
+                <Typography sx={{ mb: 2, fontWeight: 'bold' }}>Email address</Typography>
+                <TextField
+                    onChange={(e) => updateEmail(e.target.value)}
+                    id="outlined-basic"
+                    label="Outlined"
```

Comment:
label="Outlined"?
meaningful
Please change the name here to something meaningful, or since you are using a custom label in the typography above. Make the label shrink true and use <MetLabel/> in the common folder instead of the typography above. Use MetLabel as the label for any input field